source/content/partials/traffic-overages-faq.md

```diff
@@ -10,7 +10,7 @@ reviewed: ""
 
 ### Will Pantheon warn me if my site has an overage?
 
-Customers are responsible for monitoring their usage and overage risk using the Site Dashboard. Pantheon periodically reviews customer usage for overages across all plan levels and will contact you directly by email. If the change to traffic behavior exceeds your plan limit for any two months of traffic, your site will be moved to the next appropriate plan to help avoid further overages. These adjustments will be communicated in an email with a subject of "You've been right-sized".
+Customers are responsible for monitoring their usage and overage risk using the Site Dashboard. Pantheon periodically reviews customer usage for overages across all plan levels and will contact you directly by email. If the change to traffic behavior exceeds your plan limit, your site may be subject to overages enforcement. Overages protection exists for Performance and Elite sites on an annual agreement. Refer to the [FAQ](https://docs.google.com/document/d/1vaLJHXJJashwO9zsHFeywW-RQx_6uWOWH2Yr31sbQv4/edit?usp=sharing) for more details on Overage Protection and what to expect should your site go over the plan limits.
```

Comment:
Link to a public FAQ instead of a Google Doc

Reply:
Confirmed with Elaine and Michaela. There is no FAQ webpage so the approach is to add the link to the FAQ Google Doc. The link is updated: https://docs.google.com/document/d/1A6A6Jjkr_CXjda-hD0MfMqtKUkiPqS9C7Lyh6NxKH5Y/edit#heading=h.ycgc6jrgtswk

Reply:
We need to consolidate and reconcile the Google Doc FAQ with the existing [Traffic FAQ](https://docs.pantheon.io/guides/account-mgmt/traffic/faq) 

I'm working on a more thorough review round now and will likely follow up with a separate PR so that it's easier to collaborate since this PR was opened from a fork that I don't have write access to